packages/framework/presence/src/presenceStates.ts

```diff
@@ -243,7 +259,7 @@ class PresenceStatesImpl<TSchema extends PresenceStatesSchema>
 			this.props = this.nodes as unknown as PresenceStates<TSchema>["props"];
 
 			if (anyInitialValues) {
-				this.runtime.localUpdate(initial.newValues, false);
+				this.runtime.localUpdate(initial.newValues, { forceBroadcast: false });
```

Comment:
I think this is the cause of the initial signals not getting queued. The localUpdate call here needs to pass the allowable latency. Not sure where to get it in this part of the code.

Reply:
Probably would want to accumulate from the value managers as initial.newValues is accumulated in the reduction above. We can just comment for now.